src/framing/Frame.cpp

```diff
@@ -95,41 +96,85 @@ std::ostream& operator<<(std::ostream& os, ErrorCode errorCode) {
   return os << "ErrorCode(" << static_cast<uint32_t>(errorCode) << ")";
 }
 
-std::ostream& operator<<(std::ostream& os, FrameFlags frameFlags) {
-  // TODO Match the Flag names with the AllowedFlags in the Frame declarations
+std::ostream& operator<<(std::ostream& os, std::pair<FrameFlags, FrameType> frameFlagsAndType) {
```

Comment:
I still think it's better to make this a function instead of an operator<< overload, i.e. `std::ostream& writeFlags(std::ostream&, FrameType, FrameFlags);`.  Having to make a pair just to pass in two arguments is silly.